3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,161 @@
+
+//Flight Booking
+function getFullname(firstname, surname, useFormalName = false, gender = "male") {
+    if (firstname === "" || surname === "") {
+        return "";
```

Comment:
It would be beneficial to return a message prompting the user to enter a valid name instead of an empty string.